pkg/ui/src/views/reports/containers/network/index.tsx

```diff
@@ -399,20 +405,37 @@ class Network extends React.Component<NetworkProps, {}> {
     let content: JSX.Element[] = [];
     if (_.isEmpty(healthyIDs)) {
       content = [<h2>No healthy nodes match the filters</h2>];
+    } else if (latencies.length < 1) {
+      content = [<h2>Cannot show latency chart without two healthy nodes.</h2>];
     } else {
       content = this.renderLatencyTable(latencies, staleIDs, nodesSummary, displayIdentities);
     }
-
     return (
-      <div className="section">
-        <h1>Network Diagnostics</h1>
-        <NodeFilterList nodeIDs={filters.nodeIDs} localityRegex={filters.localityRegex} />
+      <div>
```

Comment:
this wrapper div is unfortunate.  we should try to upgrade to React 16 soon after 2.0, so we can use this: https://reactjs.org/blog/2017/11/28/react-v16.2.0-fragment-support.html

In this case I think returning a list would probably be ok...